### DISCUSSION TITLE
Glitches in documentation

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#168] [Developer Feedback]
There are some glitches in docs.

[push method](https://docs.zepp.com/docs/reference/device-app-api/newAPI/router/push/?_highlight=push#example)

it's not stated anywhere that referenced page must be declared in app.json
Without that push does *something* resulting black screen in simulator, and there are no exceptions, no any helpful information on what's happened.

**Comment by htoooth** [comment]
@kartun83 
谢谢，我们会在文档里面补充这方面的说明
Thank you, we will add explanations on this aspect in the document.

**Reply by kartun83** [reply]
Also, there are no details or examples how to handle *params* in push.
I found good example in https://github.com/zepp-health/zeppos-samples/blob/main/application/2.0/showcase/page/index.js , but imho it would be much useful if it would be reflected in docs.
